<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editwindow</class>
 <widget class="QWidget" name="editwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(176, 208, 255);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>781</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabFeatures">
    <attribute name="title">
     <string>Features</string>
    </attribute>
    <widget class="QGroupBox" name="gboxTimeDomin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>761</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(148, 177, 255);
background-color: rgb(221, 239, 255);</string>
     </property>
     <property name="title">
      <string>Time Domain</string>
     </property>
     <widget class="QLabel" name="labelMeanResult">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMEAN">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>MEAN</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelVarResult">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>30</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnVAR">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>VAR</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelMeanFormula">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRMS">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>127</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>RMS</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelRmsResult">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelRmsFormula">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelVarFormula">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelStdDevFormula">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>160</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelStdDevResult">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>130</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnStdDev">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>127</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Std Dev</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditTimeDomain">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>341</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(201, 255, 230);</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gboxFreqDomin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>761</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(148, 177, 255);
background-color: rgb(221, 239, 255);</string>
     </property>
     <property name="title">
      <string>Freq Domain</string>
     </property>
     <widget class="QLabel" name="labelResultFreqMean">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFreqMean">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Freq Mean</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelResultFreqPeak">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>30</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFreqPeak">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Freq Peak</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelFormulaFreqMean">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSigPower">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>127</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Sig Power</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelResultSigPower">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelFormulaSigPower">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelFormulaFreqPeak">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelFormulaSigEntropy">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>160</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelResultSigEntropy">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>130</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSigEntropy">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>127</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Sig Entropy</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditFreqDomain">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>341</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(201, 255, 230);</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabReport">
    <attribute name="title">
     <string>Report</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnSaveFeatures">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCalculate">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
